site/src/includes/components/head.njk

```diff
@@ -17,4 +17,5 @@
     {% include "script.js" %}
   {% endset %}
   <script defer>{{ js | safe }}</script>
+	<script>{% include "anchor.js" %}</script>
```

Comment:
Why use separate place for inserting the code? We already have script file for that purpose.